aws/resource_aws_ssm_patch_baseline.go

```diff
@@ -10,6 +10,38 @@ import (
 	"github.com/hashicorp/terraform/helper/validation"
 )
 
+const (
+	ssmPatchComplianceLevelUnspecified   = "UNSPECIFIED"
+	ssmPatchComplianceLevelInformational = "INFORMATIONAL"
+	ssmPatchComplianceLevelLow           = "LOW"
+	ssmPatchComplianceLevelMedium        = "MEDIUM"
+	ssmPatchComplianceLevelHigh          = "HIGH"
+	ssmPatchComplianceLevelCritical      = "CRITICAL"
+)
```

Comment:
I was thinking of the existing ones, provided by the SDK: https://github.com/terraform-providers/terraform-provider-aws/blob/master/vendor/github.com/aws/aws-sdk-go/service/ssm/api.go#L27668-L27686 😄 

and for patch OS: https://github.com/terraform-providers/terraform-provider-aws/blob/master/vendor/github.com/aws/aws-sdk-go/service/ssm/api.go#L27615-L27627

Could you make the change? last thing remaining I think! :)

Reply:
@Ninir Done!

```
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test $(go list ./... |grep -v 'vendor') -v -run=TestAccAWSSSMPatchBaseline* -timeout 120m
?   	github.com/terraform-providers/terraform-provider-aws	[no test files]
=== RUN   TestAccAWSSSMPatchBaseline_basic
--- PASS: TestAccAWSSSMPatchBaseline_basic (23.50s)
=== RUN   TestAccAWSSSMPatchBaselineWithOperatingSystem
--- PASS: TestAccAWSSSMPatchBaselineWithOperatingSystem (22.57s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	46.102s
```